From 74f4e1b30fac2bf1917f1cc8e5c55b9c1e7b2703 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 5 Oct 2022 14:14:12 +0200
Subject: [PATCH 018/529] ARM: configs: disable GCC plugins in multiv7_cleanup
 fragment

Current OSTL SDK (based on GCC 11.3.0) is not compatible with GCC plugins:
cc1: error: cannot load plugin ./scripts/gcc-plugins/arm_ssp_per_task_plugin.so: ./scripts/gcc-plugins/arm_ssp_per_task_plugin.so: undefined symbol: _ZN8opt_pass14set_pass_paramEjb
cc1: error: cannot load plugin ./scripts/gcc-plugins/arm_ssp_per_task_plugin.so: ./scripts/gcc-plugins/arm_ssp_per_task_plugin.so: undefined symbol: _ZN8opt_pass14set_pass_paramEjb
make[1]: *** [scripts/Makefile.build:117: scripts/mod/devicetable-offsets.s] Error 1

To avoid plugin load failure, GCC plugins are disabled.

Change-Id: I18860f5957e3fa6f67740ff5be4c6fd2d1657e70
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
---
 arch/arm/configs/fragment-01-multiv7_cleanup.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-01-multiv7_cleanup.config
+++ b/arch/arm/configs/fragment-01-multiv7_cleanup.config
@@ -53,6 +53,7 @@
 # CONFIG_ENABLE_MUST_CHECK is not set
 # CONFIG_ENABLE_WARN_DEPRECATED is not set
 # CONFIG_EXTCON is not set
+# CONFIG_GCC_PLUGINS is not set
 # CONFIG_GPIO_PCA953X is not set
 # CONFIG_GPIO_PCF857X is not set
 # CONFIG_I2C_DESIGNWARE_PLATFORM is not set
